src/app/admin/question/question.service.ts

```diff
@@ -0,0 +1,64 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { IQuestion, IAnswer } from './question';
+import { tap, map } from 'rxjs/operators';
+import { forkJoin, Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class QuestionService {
+
+  constructor(private http: HttpClient) { }
+
+  getTestQuestion(id: number) {
+    return this.http.get(`question/getRecordsRangeByTest/${id}/20/0`)
```

Comment:
Looks like this method always returns first 20 records. Is there some purpose for it?

Reply:
This method is related to pagination functionality, which is not done for now. I hope i would fix hard coded records per page and offset params in next iteration.